docs/en_US/reference/experiment_config.rst

```diff
@@ -0,0 +1,575 @@
+
+type: ``int``
+
+The real concurrency also depends on hardware resources and may be less than this value.
+
+
+trial gpu number
+----------------
+
+Number of GPUs used by each trial.
+
+type: ``Optional[int]``
+
+If set to zero, trials will have no access to any GPU. 
+
+If not specified, trials will be created and scheduled as if they do not use GPU,
+but they can still access all GPUs on the training machine.
+
+
+max experiment duration
+-----------------------
+
+Limit the duration of this experiment if specified.
+
+type: ``Optional[str]``
+
+format: ``number + s|m|h|d``
+
+examples: ``"10m"``, ``"0.5h"``
+
+When time runs out, the experiment will stop creating trials but continue to serve web UI.
+
+
+max trial number
+----------------
+
+Limit the number of trials to create if specified.
+
+type: ``Optional[int]``
+
+When the budget runs out, the experiment will stop creating trials but continue to serve web UI.
+
+
+nni manager ip
+--------------
+
+IP of current machine, used by training machines to access NNI manager. Not used in local mode.
+
+type: ``Optional[str]``
+
+If not specified, this will be the default IPv4 address of outgoing connection.
+
+// there is an easy way to get the IP mentioned above, I will remove netifaces denendency
+
+
+use annotation
+--------------
+
+Enable `annotation <https://nni.readthedocs.io/en/stable/Tutorial/AnnotationSpec.html>`_.
+
+type: ``bool``
+
+default: ``False``
+
+When using annotation, `search space`_ and `search space file`_ should not be specified manually.
+
+
+debug
+-----
+
+Enable debug mode.
+
+type: ``bool``
+
+default: ``False``
+
+When enabled, logging will be more verbose and some internal validation will be loosen.
+
+
+log level
+---------
+
+Set log level of whole system.
+
+type: ``Optional[str]``
+
+values: ``"trace"``, ``"debug"``, ``"info"``, ``"warning"``, ``"error"``, ``"fatal"``
+
+Defaults to "info" or "debug", depending on `debug`_ option.
+
+Most modules of NNI will be affected by this value, including NNI manager, tuner, training service, etc.
+
+The exception is trial, whose logging level is directly managed by trial code.
+
+For Python modules, "trace" acts as ``logging.DEBUG`` and "fatal" acts as ``logging.CRITICAL``.
+
+
+experiment working directory
+----------------------------
+
+Specify the `directory`_ to place log, checkpoint, metadata, and other run-time stuff.
+
+type: ``Optional[str]``
+
+By default uses ``~/nni-experiments``.
+
+NNI will create a subdirectory named by experiment ID, so it is safe to use same directory for multiple experiments.
+
+
+tuner gpu indices
+-----------------
+
+Limit the GPUs visible to tuner, assessor, and advisor.
+
+type: ``Optional[Union[list[int], str]]``
+
+This will be the ``CUDA_VISIBLE_DEVICES`` environment variable of tuner process.
+
+Because tuner, assessor, and advisor run in same process, this option will affect them all.
+
+
+tuner
+-----
+
+Specify the tuner [TODO:link]
```

Comment:
Can we have the link now?

Reply:
I can't find a doc mention built-in tuner and custom tuner at same time.